tools/populate/populate-images.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+# Populate an Oxide lab host running Omicron with images from server catacomb.
```

Comment:
Would it be possible to do a `set -eu` for this script?

As an example, if someone tries running without the `oxide` CLI installed, it'll keep trying to do work, even after multiple failures.

Reply:
Done.